webapp/src/utils/styles.js

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {changeOpacity} from 'mattermost-redux/utils/theme_utils';
+
+export const getStyleForReactSelect = (theme) => {
+    if (!theme) {
+        return null;
+    }
+
+    return {
+        menuPortal: (provided) => ({
+            ...provided,
+            zIndex: 9999,
+        }),
+        control: (provided, state) => ({
+            ...provided,
+            color: theme.centerChannelColor,
+            background: theme.centerChannelBg,
+
+            // Overwrites the different states of border
+            borderColor: state.isFocused ? changeOpacity(theme.centerChannelColor, 0.25) : changeOpacity(theme.centerChannelColor, 0.12),
+
+            // Removes weird border around container
```

Comment:
```suggestion
            // Removes border around container
```

it could be `weird` according to us, but we should not write that